internal/tracking/tracking_mock.go

```diff
@@ -28,13 +28,32 @@ type EventTrackerMock struct {
 
 func (m *EventTrackerMock) AddDefaultMocks() {
 	m.On("FlushToLogstash", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(nil)
```

Comment:
```suggestion
```

🪓 Ignore this suggestion but I'm now wondering if we can be confident that we're setting up all mock implementations within tests for this method, so a default is no longer needed?

Reply:
Sure, I'm okay with that. We should think about how we want to approach the `.DefaultMocks()`. I think it saves us a lot of scaffolded code, but sometimes comes off as magical and leads to brittle tests that dependent on the default mocks. 🤔 

However, since `FlushToLogstash` is only used on `root.go`, I think it's a safe one to always mock explicitly.

Commit 5128747 removes the default mock and declares it explicitly in the `root_test.go` table test.